weed/remote_storage/gcs/gcs_storage_client.go

```
@@ -71,15 +106,14 @@ func (s gcsRemoteStorageMaker) Make(conf *remote_pb.RemoteConf) (remote_storage.
 				return nil, fmt.Errorf("failed to read credentials file %s: %w", googleApplicationCredentials, err)
 			}
 		}
-		creds, err := google.CredentialsFromJSON(context.Background(), data, storage.ScopeFullControl)
+		creds, err := google.CredentialsFromJSON(ctx, data, storage.ScopeFullControl)
```

golangci-lint (2.12.2)

[error] SA1019: google.CredentialsFromJSON is deprecated: This function is deprecated because of a potential security risk. It does not validate the credential configuration. The security risk occurs when a credential configuration is accepted from a source that is not under your control and used without validation on your side.

(staticcheck)